rpc/query_service.go

```diff
@@ -226,6 +227,8 @@ func MakeUnsafeQueryServiceHandler(logger log.TMLogger) http.Handler {
 	routes["unsafe_stop_cpu_profiler"] = rpcserver.NewRPCFunc(rpccore.UnsafeStopCPUProfiler, "")
 	routes["unsafe_write_heap_profile"] = rpcserver.NewRPCFunc(rpccore.UnsafeWriteHeapProfile, "filename")
 
+	routes["account_balances"] = rpcserver.NewRPCFunc(svc.GetAccountBalance, "contract_address")
```

Comment:
-> `rpcserver.NewRPCFunc(svc.GetAccountBalance, "contracts")`

Which should then be queried like so:
```
curl 'localhost:26657/accounts_balances?contracts=["eth","loom","0xdeadbeef"]'
```